registry/spec.md

```diff
@@ -682,8 +682,14 @@ Response:
 HTTP/1.1 ...
 Content-Type: application/json; charset=utf-8
 Content-Length: nnnn
+{
```

Comment:
Can you add a blank line before this line?

Reply:
There's already a blank line between the headers and the body... is there a reason we'd need another?

Reply:
the comment is old. he added the line